cmake/FindCMSIS.cmake

```diff
@@ -7,6 +9,32 @@ if(STM32H7 IN_LIST CMSIS_FIND_COMPONENTS)
 endif()
 list(REMOVE_DUPLICATES CMSIS_FIND_COMPONENTS)
 
+foreach(COMP ${CMSIS_FIND_COMPONENTS})
```

Comment:
I'm not expert so I'm sorry if it's supposed to be trivial for me. But if you find it relevant to beginners and maintainers: please add (relevant) comments into the code.

Reply:
I added more documentation for all component list filler loops